scripts/run-bundled-codegen.sh

```diff
@@ -0,0 +1,64 @@
+# Bail if errors occur
+set -e
```

Comment:
This is often a best practice which takes this a step further:

```suggestion
# Exit on all errors, undeclared variables and pipefailures.
set -euo pipefail
```

This isn't POSIX, but it is compatible with `bash` and `zsh`, and `fish`.